index.js

```diff
@@ -23,22 +24,22 @@ module.exports = (PluginHost) => {
    *    url: string
    *  }
    */
-  app.options.addDeclaration({ name: 'links', type: 'Array' })
+  app.options.addDeclaration({ name: 'links', type: ParameterType.Mixed })
```

Comment:
Is this still an array type?

Reply:
Yes, it is. But it deal with Array in different behavior as we think:
```js
        case ParameterType.Array:
            if (Array.isArray(value)) {
                return Result.Ok(value.map(String));
            } else if (typeof value === 'string') {
                return Result.Ok(value.split(','));
            }
            return Result.Ok([]);
```
It will transform all the elements in array into Array<string>...
But for the `Mixed` type, it will keep what it should be:
```js
        case ParameterType.Mixed:
            return Result.Ok(value);
```